designer/formatStagedFrontendFiles.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
```

Comment:
This script doesn't change current directory and uses paths relative to directory root, so it will fail when called as `./formatStagedFrontendFiles.sh`. Is this ok?